### PR TITLE
export default server capabilities

### PR DESCRIPTION
**Pull request by jlipps**
so we can use them in xcuitest too.

should assist with https://github.com/appium/appium/issues/6771

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -43,7 +43,6 @@ let serverCapabilities = {
 class IosDriver extends BaseDriver {
   resetIos () {
     this.appExt = ".app";
-    _.defaults(this.opts, serverCapabilities);
```
Comment (jlipps) [inline lib/driver.js]: i couldn't think of any reason we actually wanted these on `this.opts` other than to return them, which we do by extending down below. so removing for sanity. but it's possible these are needed somehow. cc @imurchie

Comment (imurchie) [inline lib/driver.js]: Yeah, I don't think we make use of these.

**Comment by imurchie** [comment]
Looks good to me! It seems to have gotten through the build once, during a window when Travis worked.

I'm going to merge so we can get the `appium-xcuitest-driver` fix through.

**Comment by imurchie** [comment]
Published in `appium-ios-driver@1.12.20`

**Pull request merged**